sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -713,6 +751,14 @@ export interface FreshnessScoringParameters {
     boostingDuration: string;
 }
 
+// @public (undocumented)
+export interface GenerativeQueryRewrites {
+    // (undocumented)
+    count?: number;
+    // (undocumented)
+    rewritesType: "generative";
```

Comment:
why is this field needed? Is it part of a discriminated union? If not, maybe we dont need it?